app/code/Magento/Customer/etc/adminhtml/system.xml

```diff
@@ -86,6 +86,11 @@
                     <comment>To show VAT number on Storefront, set Show VAT Number on Storefront option to Yes.</comment>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="email_required_create_order" translate="label comment" type="select" sortOrder="58" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Email is required field for Admin order creation</label>
```

Comment:
Please add translatable fields value to the i18n/en_US.csv

Reply:
Hi @swnsma ,
Thank you for your review, I am working in on it.

Reply:
@swnsma  

Done.

Please view the attachment for more details.

![language_translation](https://user-images.githubusercontent.com/16253886/64530455-c8a73800-d32a-11e9-9924-c761ad8462f4.png)

Reply:
@solwininfotech,
Could I ask you to discard all changes from app/code/Magento/Customer/i18n/en_US.csv which are not related to this PR?
Also, as we are going to locate the setting in Magento_Sales, please move translations to app/code/Magento/Sales/i18n/en_US.csv

Reply:
@swnsma 

Done.

Please review it.